_episodes/05-github-pages.md

```diff
@@ -118,11 +118,12 @@ Usually it's available instantly, but it can take a few seconds and in the worst
 > Instead of making edits on the GitHub website you can 'clone' the fork to your local machine
 > and work there.
 >
-> Try following the rest of the steps under "Time to Submit Your First PR"
-> at this guide: <https://www.thinkful.com/learn/github-pull-request-tutorial/Writing-a-Good-Commit-Message#Time-to-Submit-Your-First-PR>
+> Try following the steps in this guide:
+> <https://docs.github.com/en/get-started/quickstart/contributing-to-projects>
```

Comment:
Another point to add to https://github.com/LibraryCarpentry/lc-git/pull/131#pullrequestreview-1147654948: [long links are an accessibility issue](https://webaim.org/techniques/hypertext/link_text#link_length):

> Remember that blind users cannot visually skim through lengthy links. They must listen to the entire text. Some get frustrated with long link text and impatiently move on to the next link if they cannot understand the purpose of the link after the first few words.

```suggestion
> Try following the steps in [GitHub's quickstart guide for contributing to projects](https://docs.github.com/en/get-started/quickstart/contributing-to-projects)
```